UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/DataHelper.cs

```diff
@@ -43,7 +43,7 @@ public List<string> GetReissueProducts()
 
         public List<string> GetProductIdentifiersForLargeMedia()
         {
-           return new List<string>() { "DE5NOBRK", "DE4NO18Q", "DE416080", "DE260001" };
+           return new List<string>() { "FR570300", "SE6IIFE1", "NO3B2020", "GB20486A", "RU3P0ZM0", "US5CN13M", "CA172005", "DE521900", "NZ300661", "KR676D03" };
```

Comment:
Hi Raymond, If we pass any incorrect or duplicate product along with other valid products, it will return the data for the valid products(200 status code) and will show the message in response about the incorrect or duplicate products.
I have added the snapshot, please have a look.

![image](https://user-images.githubusercontent.com/85937324/184109449-84b94172-a170-4901-bc3d-61be065955fa.png)
![image](https://user-images.githubusercontent.com/85937324/184109551-8b6ecc2f-ab64-4a1e-8857-fc08eb14d8e5.png)

Reply:
have you added tests to capture this unhappy path flow

Reply:
Hi Raymond, Yes this has been already covered in the **ExchangeSetByProductIdentifiers.cs** and executed in pipeline as well.
Below, are the test name of those:
WhenICallTheApiWithValidAndInvalidProductIdentifiers_ThenACorrectResponseIsReturned
WhenICallTheApiWithDuplicateProductIdentifiers_ThenACorrectResponseIsReturned
Thanks,